webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/SettingsApi.scala

```diff
@@ -124,8 +122,18 @@ class SettingsApi(
 
   val allSettings_v8 = RestUseReverseDNS :: allSettings_v10
 
+  val allSettings_v12 =   RestReportProtocolDefault :: RestSyslogProtocolDisabled :: allSettings_v10
```

Comment:
why is it based on 10 and not 11?

Reply:
Because version 11 did not change anything in settings?

Reply:
exactly, latest API settings is v10 (v11 is simply forwarding to v10)